src/components/ObsDetails/Attribution.js

```diff
@@ -13,25 +14,25 @@ type Props = {
 // https://github.com/inaturalist/inaturalist/blob/768b9263931ebeea229bbc47d8442ca6b0377d45/app/webpack/shared/components/observation_attribution.jsx
 const Attribution = ( { observation }: Props ): Node => {
   const { user } = observation;
-  const licenseCode = observation.license_code;
+  const licenseCode = _.upperCase( observation.license_code );
   const copyrightAttribution = user ? ( user.name || user.login ) : t( "unknown" );
 
   const renderLicenseCode = ( ) => {
     if ( !licenseCode ) {
       return t( "all-rights-reserved" );
     } if ( licenseCode === "cc0" ) {
-      return t( "no-rights-reserved" ) + licenseCode;
+      return `${t( "no-rights-reserved" )} ${licenseCode}`;
     }
-    return t( "some-rights-reserved" ) + licenseCode;
+    return `${t( "some-rights-reserved" )} ${licenseCode}`;
```

Comment:
This entire string needs to be localized because the ordering might different in different locales, so it should look something like `t( "attribution_cc_by_nc" )`.